CHANGELOG.rst

```diff
@@ -282,6 +282,16 @@ Changed
 
 * Update orquesta to v1.4.0.
 
+* Modified action delete api. Action delete api removes related action/workflow files on disk
```

Comment:
This has been added to the changes made to 3.5.0. It should be in the "in development" section at the top.